common/libs/values.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import type Big from 'big.js';
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
Lets ship with bn.js since it is what ethereumjs-* uses. It can read/write hex, but doesnt support decimals so displaying code should be adjusted to account for that